README.md

```diff
@@ -1,104 +1,74 @@
+- `function/services/ssm` - Supports integration with AWS SSM.
 - `template.yml` - An AWS CloudFormation template that creates an application.
-- `1-create-bucket.sh`, `2-deploy.sh`, etc. - Shell scripts that use the AWS CLI to deploy and manage the application.
-
-Variants of this sample application are available for the following languages:
-
-- Python – [blank-python](/sample-apps/blank-python).
-- Ruby – [blank-ruby](/sample-apps/blank-ruby).
-- Java – [blank-java](/sample-apps/blank-java).
-- Go – [blank-go](/sample-apps/blank-go).
-- C# – [blank-csharp](/sample-apps/blank-csharp).
-- PowerShell – [blank-powershell](/sample-apps/blank-powershell).
 
-Use the following instructions to deploy the sample application. For an in-depth look at its architecture and features, see [Blank Function Sample Application for AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/samples-blank-nodejs.html) in the developer guide.
 
 # Requirements
-- [Node.js 10 with npm](https://nodejs.org/en/download/releases/)
+- [Node.js 14.6 or later with npm](https://nodejs.org/en/download/releases/)
 - The Bash shell. For Linux and macOS, this is included by default. In Windows 10, you can install the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to get a Windows-integrated version of Ubuntu and Bash.
-- [The AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) v1.17 or newer.
+- [Oracledb for nodejs](https://node-oracledb.readthedocs.io/en/latest/) - Will need to follow step 2.3 onwards to set this up.
 
-If you use the AWS CLI v2, add the following to your [configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html) (`~/.aws/config`):
+Not mandatory but useful if using VSCode:
+- Prettier formatter extension
+- AWS Toolkit extension
 
-```
-cli_binary_format=raw-in-base64-out
-```
+# Local development setup
 
-This setting enables the AWS CLI v2 to load JSON events from a file, matching the v1 behavior.
-
-# Setup
 Download or clone this repository.
+Add an .env file containing:
 
-    $ git clone https://github.com/awsdocs/aws-lambda-developer-guide.git
-    $ cd aws-lambda-developer-guide/sample-apps/blank-nodejs
-
-To create a new bucket for deployment artifacts, run `1-create-bucket.sh`.
+   `NODE_ENV = 'local'`
 
-    blank-nodejs$ ./1-create-bucket.sh
-    make_bucket: lambda-artifacts-a5e491dbb5b22e0d
+Configure local code:
+- In `function/index.test.js` unskip the `'Should run the function handler'` test
+- In `function/db/dbPooljs` and `function/db/index.js` uncomment the lines starting with `oracledb.initOracleClient`
+- Comment lines 57 - 66 in `function/index.js`
 
-To build a Lambda layer that contains the function's runtime dependencies, run `2-build-layer.sh`. Packaging dependencies in a layer reduces the size of the deployment package that you upload when you modify your code.
+Configure local AWS environment:
```

Comment:
Did you never get localstack setup? The benefit is you could include a Makefile that defines all these steps in one nice command so that developers can just run 'make init' or similar and have all the resources setup themselves rather than having to create an AWS account and do all these things. If you've not got the time before going on leave that's okay we can get it sorted but it would be my preference to do this so we don't have to worry about everyone coming onto the project (including CICA if they ever wanted to make changes themselves) having to setup personal AWS accounts and manage them etc.

Reply:
I'll speak to Adam about this and see if I can get it set up quickly